docs/modules/core/pages/vr/virtualrealitycontrollers.adoc

```diff
@@ -0,0 +1,125 @@
+= Virtual Reality Controllers
+:revnumber: 1.0
+:revdate: 2021/12/29
+
+Having successfully seen a VR cube in the xref:vr/virtualreality.adoc[Virtual reality hello world] you'll want to put your hands in the world so you can start interacting. 
+
+== Where are we, what are we pointing at
+
+Be aware that the controllers positions and rotations are in world coordinates, not relative to the camera
+
+To get the number of controllers you'll want
```

Comment:
Might be just me, but for some reason tech documentation telling me what I'll want to do sounds wrong to me. I'd probably write it more in the style of "To get the number of controllers, use the following:". It sounds much more tone neutral IMO. This would also apply in other instances where "you'll want to do X" is used in this PR.

What do the others think?

Reply:
Fair, my style is perhaps a little too conversational. I've toned it down a bit